### DISCUSSION TITLE
Illegal offset type in livewire form objects

### DISCUSSION THREAD

**Discussion by fanorama** [livewire/livewire#6542] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. create form object
2. create a properties that hold an array
3. create the rule

 #[Rule([
        'form.items' => 'required|array',
        'form.items.*.keterangan' => 'required|min:5',
        'form.items.*.customer' => 'required|exists:customers,id',
        'form.items.*.project' => 'required|exists:projects,id',
        'form.items.*.amount' => 'required|int',
    ], attribute: [
        'form.items' => 'items',
        'form.items.*.keterangan' => 'keterangan',
        'form.items.*.customer' => 'customer',
        'form.items.*.project' => 'project',
        'form.items.*.amount' => 'amount',
    ], message: [
        'form.items.required' => 'The :attribute required',
        'form.items.*.keterangan.min' => 'The minimal character :attribute is 5',
        'form.items.*.customer.exists' => 'The :attribute is invalid',
        'form.items.*.project.exists' => 'The :attribute is invalid',
    ])]
    public array $items = [];

![Screenshot_2023-08-26-22-18-13_1920x1080](https://github.com/livewire/livewire/assets/60024110/5e53992b-9b7e-4b36-9d6b-f1c2fe6b33e3)

I get this result error:

![Screenshot_2023-08-26-22-19-21_1920x1080](https://github.com/livewire/livewire/assets/60024110/e50299ae-02cc-4c06-993c-87e523153903)

when I commented out my message arguments the error is gone.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Jaiieq** [comment]
Add 'translate: false' to your rule.  It should fix the Illegal offset type until the bug is fixed in 3.0 stable.  I ran into the same issue.
